multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/api/impl/OperationsApiServiceImpl.java

```diff
@@ -139,9 +141,14 @@ public ResponseEntity<List<Log>> getOperationLogs(String spaceGuid, String opera
     public ResponseEntity<String> getOperationLogContent(String spaceGuid, String operationId, String logId) {
         try {
             operationsApiServiceAuditLog.logGetOperationLogContent(SecurityContextUtil.getUsername(), spaceGuid, operationId, logId);
-            String content = logsService.getLogContent(spaceGuid, operationId, logId);
+            String content = logsService.getOperationLog(spaceGuid, operationId, logId);
+
+            if (content == null || content.isEmpty() ||  content.isBlank()) {
```

Comment:
isEmpty and isBlank?
-> won't we drop the columns with the liquibase changes? is this backwards compatible?